src/DurableTask/DurableTaskClient.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using DurableTask.Core;
```

Comment:
If possible, I'd like to avoid exposing DurableTask.Core types in the public API.